app/src/androidTest/java/com/rkm/tasky/database/EventTest.kt

```diff
@@ -96,6 +95,173 @@ class EventTest {
         assertTrue(result != null)
     }
 
+    @Test
+    fun insertEventDetails() = runTest {
+        eventDataSource.upsertAllEventInfo(
+            event = event,
+            attendees = attendees,
+            photos = photos,
+            photoDao = photoDataSource,
+            attendeeDao = attendeeDataSource
+        )
+
+        val eventResult = eventDataSource.getEventById(event.id)
+        val attendeesResult = attendeeDataSource.getAttendeesByEventId(listOf(event.id))
+        val photosResult = photoDataSource.getPhotosByEventId(listOf(event.id))
+
+        assertEquals(eventResult, event)
+        assertEquals(attendees.toHashSet(), attendeesResult.toHashSet())
+        assertEquals(photos.toHashSet(), photosResult.toHashSet())
+    }
```

Comment:
Does inserting event details involve any significant logic? If it's really just a call to your DAO, there's no real point in testing that since you effectively test the Room library here, but not your code. That's the job from the library authors